README.md

```diff
@@ -13,8 +13,8 @@
 [![Fedora package](https://img.shields.io/fedora/v/nest?logo=fedora)](https://src.fedoraproject.org/rpms/nest)
 [![Conda version](https://img.shields.io/conda/vn/conda-forge/nest-simulator.svg?logo=conda-forge&logoColor=white)](https://anaconda.org/conda-forge/nest-simulator)
 [![Homebrew version](https://img.shields.io/homebrew/v/nest.svg?logo=apple)](https://formulae.brew.sh/formula/nest)
-[![Docker Image Version](https://img.shields.io/docker/v/nestsim/nest?label=docker&sort=semver&logo=docker&logoColor=white)](https://hub.docker.com/r/nestsim/nest)
-[![Virtual applicance](https://img.shields.io/badge/VM-v3.1-blue?logo=CodeSandbox)](https://nest-simulator.readthedocs.io/en/latest/download.html#download-livemedia)
+[![Docker Image Version](https://img.shields.io/badge/docker-v3.4-blue?logo=docker)](https://docker-registry.ebrains.eu/harbor/projects/6/repositories/nest-simulator)
```

Comment:
Why did the URL change to a non-public site? Which seems to be broken, btw.
Even if not built on dockerhub, is it not possible to push the image to dockerhub somehow?

Reply:
I had linked to our main repository. But you are right that this overview page is not very useful. I'm not a fan of duplicating data, but I still updated Docker Hub with the NEST 3.4_rc1 image. The link points to the tag page, as I think this is the most concise one.